jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -284,6 +284,17 @@ def step080_customize_java_args(script_dir, server_java_args, server_era) {
     }
 }
 
+def step081_customize_jruby_jar(script_dir, jruby_jar, server_era) {
+    if ((jruby_jar == null) || (jruby_jar == "")) {
+        echo "Skipping jruby jar configuration because none specified in job"
```

Comment:
It might be good to go ahead and set the `JRUBY_JAR` variable to empty in the sysconfig file even if the configuration didn't specify a value for it.  For cases where you might be doing iterative work where you flip back and forth between `JRUBY_JAR` values on the same VM, it might be good to have the piece of mind that the value would be set from 9k back to the default in the case you didn't reinstall server components between test runs.  A value of `JRUBY_JAR=` in the sysconfig file is effectively the same as not having it there at all and for versions of puppetserver that don't look at the variable, having it there shouldn't be a problem.